### ISSUE TITLE
temperature value with one decimal place after decimal point

### ISSUE THREAD

**Issue by milanissimo1989** [johnwalicki/node-red-contrib-twc-weather#8]
Hello, is there a possibility that the temperature value is read out with one decimal place after the decimal point? Sorry for my bad english.

metric: object
temp: 3             -----> e.g. 3,1 ?
heatIndex: 3
dewpt: 2
windChill: 3
windSpeed: 3
windGust: 5
pressure: 994.14
precipRate: 0
precipTotal: 1.6
elev: 423

**Comment by johnwalicki** [comment]
The Node-RED node just passes thru the values from the TWC api call. It doesn't modify the value or add decimal places.  Which node are you using in your flow?

**Comment by milanissimo1989** [comment]
hey i using this one "pws observations"

**Comment by johnwalicki** [comment]
Hi - Glad that you find the TWC PWS Node-RED nodes helpful.
Here is the TWC API Documentation Link : https://ibm.co/v2PWSCC
The Node-RED `PWS Observations` node calls this API.
If you want to add a decimal to a whole number, use `.toFixed(1)`
```
var formatted_temp = msg.payload.observations[0].metric.temp.toFixed(1);
```

**Comment by johnwalicki** [comment]
@milanissimo1989 - Did the above suggestion work?  I'd like to close this issue.

**Comment by milanissimo1989** [comment]
Where do I enter the line? In a function node? Unfortunately, I am only a beginner.

**Comment by jimison** [comment]
Update: added numericPrecision API feature in Pull Request.

In the TWC API Documentation:

![image](https://user-images.githubusercontent.com/19612746/160299421-4c0a90c3-75c7-47ca-93f9-dea7cd9d6ad5.png)

I've confirmed that by appending my URL to include "&numericPrecision=decimal", several values are returned with a fixed tenths place instead of just integers. The affected values are as follows:

- humidity
- temp
- heatIndex
- dewpt
- windChill
- windSpeed
- windGust
- elev

All other values remain unchanged.

Adding this as an Numeric Precision dropdown with options of Integer or Decimal to the API call should suffice. If Integer is selected, leave the API request as is. If Decimal is selected, append the request with "&numericPrecision=decimal".

Hope this helps!

**Comment by jimison** [comment]
Created pull requests for the changes to pws-current.js and pws-current.html (sorry for the separate requests).

**Comment by johnwalicki** [comment]
Excellent - Thanks @jimison  - Closing this issue. Fixed by #10 
I will bundle a node-red-contrib-twc-weather 0.5.0 and push to npmjs.
